base_restrict_user_fields/models/res_users.py

```diff
@@ -0,0 +1,15 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    unrestrict_fields = fields.Boolean(string="Don't restrict fields", compute="_compute_unrestrict_fields")
+
+    def _compute_unrestrict_fields(self):
+        for user in self:
+            user.unrestrict_fields = user.has_group("base_restrict_user_fields.group_user_mgmt_unrestrict_fields")
```

Comment:
```suggestion
            user.unrestrict_fields = self.env.user.has_group("base_restrict_user_fields.group_user_mgmt_unrestrict_fields")
```
@dalonsod Shouldn't it be like this?

Reply:
Tested current implementation, I think you're right. Please @Anxo82 make the suggested change, but calculating once `self.env.user.has_group("base_restrict_user_fields.group_user_mgmt_unrestrict_fields")`, outside `for` statement, because its calculation doesn't depend on `user`.